client-js/app/elements/labrad-nodes.html

```diff
@@ -180,9 +190,40 @@
     td.controls {
       width: 350px;
     }
+    #buttons {
+      height: 50px;
+    }
+    #buttons paper-icon-button {
+      margin: 5px;
+    }
+    .autostartFilterOn {
+      color: #000;
+    }
+    .autostartFilterOff {
+      color: #aaa;
+    }
   </style>
   <template>
-    <div>
+   <div id="container" class='fit'>
+   <paper-header-panel id="left-column">
+    <div class="paper-header" id="buttons">
+      <paper-icon-button
+        hidden$={{isAutostartFiltered}}
+        class='autostartFilterOff'
+        icon="stars"
+        id="star"
+        on-click="autostartFilter"
+        title="Show Only Autostart Servers"></paper-icon-button>
+      <paper-icon-button
```

Comment:
A helper icon/button would be very good, especially for setting `title` text. I'm not happy with my `hidden` hacks, but it was the nicest way I could find to do it without breaking the separation of script and style.